src/app/financial-chart/axis-types/financial-chart-axis-types.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangeDetectionStrategy, Component } from "@angular/core";
+import { StockDataService } from "../services/stock-data.service";
+
+import { IgxPercentChangeYAxisComponent
+} from "igniteui-angular-charts/ES5/igx-percent-change-y-axis-component";
+import { IgxPercentChangeYAxisDynamicModule
+} from "igniteui-angular-charts/ES5/igx-percent-change-y-axis-dynamic-module";
+import { IgxPercentChangeYAxisModule
```

Comment:
You were supposed to import this into the samples module and add it to the imports array...

Reply:
Why do you even have any of these modules imported here?